### DISCUSSION TITLE
Theme DaisyUi with Tauri

### DISCUSSION THREAD

**Discussion by vincehi** [saadeghi/daisyui#2772] [Q&A]
Hello, I built an application with Tauri and overall it works correctly except that I cannot override the theme in the Tailwind.config. On the other hand, if I display it in a browser rather than in Tauri, it works for the same instance. Do you have any idea why this isn't working?
https://github.com/bazalp/pulp
<img width="1440" alt="Capture d’écran 2024-01-18 à 08 52 35" src="https://github.com/saadeghi/daisyui/assets/25566437/10333294-18d2-42a2-865f-d2684273e0d1">

right is Tauri - tab is blue
left is Arc (same in safari and other) - tab is orange

**Comment by vincehi** [comment]
With v3 it's ok

**Comment by saadeghi** [comment]
I don't have experience with Tauri.  

Looks like the browser engine it's using doesn't support OKLCH colors.

- Is this the latest version of Tauri?
- Which browser engine is it using? Which version?

**Comment by hut36** [comment]
Same problem here. See https://github.com/saadeghi/daisyui/issues/2703#issuecomment-1884246271 for the reason. I can confirm that changing from oklch(0.93 0 0) to oklch(93% 0 0) works.

```
$ awk '/CFBundleVersion/{getline;gsub(/<[^>]*>/,"");print}' /System/Library/Frameworks/WebKit.framework/Resources/Info.plist
        17613.3.9.1.16
```


```
[✔] Environment
    - OS: Mac OS 12.7.2 X64
    ✔ Xcode Command Line Tools: installed
    ✔ rustc: 1.73.0 (cc66ad468 2023-10-03)
    ✔ cargo: 1.73.0 (9c4383fb5 2023-08-26)
    ✔ rustup: 1.26.0 (5af9b9484 2023-04-05)
    ✔ Rust toolchain: stable-x86_64-apple-darwin (default)
    - node: 20.4.0
    - pnpm: 8.14.3
    - yarn: 1.22.19
    - npm: 9.7.2
    - bun: 1.0.4

[-] Packages
    - tauri [RUST]: 1.5.4
    - tauri-build [RUST]: 1.5.1
    - wry [RUST]: 0.24.7
    - tao [RUST]: 0.16.6
    - tauri-cli [RUST]: 1.5.9
    - @tauri-apps/api [NPM]: 1.5.3
    - @tauri-apps/cli [NPM]: 1.5.9

[-] App
    - build-type: bundle
    - CSP: unset
    - distDir: ../dist
    - devPath: http://localhost:1420/
    - framework: Svelte
    - bundler: Vite
```

edit: typo
